tox.ini

```
@@ -0,0 +1,21 @@
+[tox]
+requires =
+	tox>=4
```

```suggestion
requires =
    tox>=4
```
4 space instead of tab indent